web/.eslintrc.js

```diff
@@ -93,14 +82,10 @@ module.exports = {
     'jsx-a11y/no-static-element-interactions': 'off',
 
     // Rules we want to enable one day
-    'curly': 'off',
     'no-nested-ternary': 'off',
-    'max-len': ['off', {'code': 120}],
     'no-underscore-dangle': 'off',
-    'react/jsx-one-expression-per-line': 'off',
```

Comment:
Removes style-related rules, dunno if this conflicts with your changes @Kongkille - otherwise we can remove them here

Reply:
I did have curly enabled on mine but we could just remove it

Reply:
I did add curlies to the code, did not realize it would be part of prettier :)